.travis.yml

```diff
@@ -6,6 +6,10 @@ jdk:
 before_install:
 - mvn clean
 - git clone -b travis `git config --get remote.origin.url` target/travis
+- openssl aes-256-cbc -K $encrypted_631490ecae8f_key -iv $encrypted_631490ecae8f_iv -in target/travis/signing-tools.tar.enc -out target/travis/signing-tools.tar -d
+- mkdir target/travis/signing-tools
+- chmod 700 target/travis/signing-tools
```

Comment:
I added that line after `gpg` complained that the folder containing the keys was readable by more than the user.